rotkehlchen/db/updates.py

```diff
@@ -0,0 +1,113 @@
+import json
+import logging
+import sys
+from typing import TYPE_CHECKING, Any
+
+import requests
+
+from rotkehlchen.assets.spam_assets import update_spam_assets
+from rotkehlchen.constants.timing import DEFAULT_TIMEOUT_TUPLE
+from rotkehlchen.errors.misc import RemoteError
+from rotkehlchen.globaldb.handler import GlobalDBHandler
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.utils.mixins.serializableenumvalue import SerializableEnumValueMixin
+from rotkehlchen.utils.network import query_file
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.dbhandler import DBHandler
+    from rotkehlchen.user_messages import MessagesAggregator
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+LAST_DATA_UPDATES_KEY = 'last_data_updates_ts'
+SPAM_ASSETS_URL = 'https://raw.githubusercontent.com/rotki/data/{branch}/updates/spam_assets/assets_v{version}.json'  # noqa: E501
+
+
+class UpdateType(SerializableEnumValueMixin):
+    SPAM_ASSETS = 'spam_assets'
+    RPC_NODES = 'rpc_nodes'
+    CONTRACTS = 'contracts'
```

Comment:
As I explained later this is not gonna work. Create a proper class and use it properly below

```suggestion
class UpdateType(Enum):
    SPAM_ASSETS = 'spam_assets'
    RPC_NODES = 'rpc_nodes'
    CONTRACTS = 'contracts'
    
    def serialize(self) -> str
        """Serializes the update type for the DB and API"""
        return f'{self.value}_last_update_ts'
        
    @classmethod
    def deserialize(cls: type[T], value: str) -> T:
    """Deserialize string from api/DB to UpdateType
    May raise:
    - Deserialization error if value is not a valid UpdateType
    """
    try:
        return cls(value[:-15])  # length of the last_update_ts suffix
    except ValueError as e:
        raise DeserializationError(f'Failed to deserialize UpdateTypevalue {value}') from e
```

Reply:
okey but we don't store the timestamp, is the version